src/components/home/Events.jsx

```diff
@@ -6,15 +6,44 @@ import Link from '@mui/material/Link';
 import Paper from '@mui/material/Paper';
 import Typography from '@mui/material/Typography';
 
+const mobileStyle = {
```

Comment:
I don't think either of these are doing anything so they can be removed.

Reply:
are you referring to lines 6 and 7? I may be wrong but I believe they are necessary. I've tried removing them and the page crashes. We are using the Paper and Typography components from mui, so its just a simple import.  Let me know if I'm misunderstanding.

Reply:
the tag for this starts on line 9. was referring to `mobileStyle` and `desktopStyle`. The imports are necessary.

Reply:
oh gotcha, okay, i've removed the unneeded lines, thank you.